pkg/api/documentsschema/documentsschema.proto

```diff
@@ -0,0 +1,252 @@
+syntax = "proto3";
+
+package immudb.documentsschema;
```

Comment:
minor (not required) but it's rare to me `documentsschema`, probably `documentschema` or under `pkg/api/document` or similar thing could be simpler

Reply:
Resolved